src/base/bittorrent/session.h

```diff
@@ -285,6 +285,7 @@ namespace BitTorrent
         void torrentsUpdated();
         void addTorrentFailed(const QString &error);
         void torrentAdded(BitTorrent::TorrentHandle *const torrent);
+        void torrentNew(BitTorrent::TorrentHandle *const torrent);
```

Comment:
Would not it be more correct to use `newTorrent` signal name?
Although I still tend to rename the old signal.